CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## Unreleased
+IMPROVEMENTS:
```

Comment:
Should we go ahead and add the version header for the version we are releasing? That way we don't need to open another PR